lib/pltr/v2/selectors/tagsThirdOrder.js

```diff
@@ -64,9 +66,21 @@ export const tagsFilterItemsSelector = createSelector(
         return filteredItems
       }
       case 'characters': {
-        const filteredItems = tags.filter((tag) =>
-          characters.find((char) => char.tags?.includes(tag.id))
-        )
+        const filteredItems = tags.filter((tag) => {
```

Comment:
Can we have a test for this, too, please? :)

Reply:
Added :)